testsuite/utils/src/main/resources/META-INF/keycloak-server.json

```diff
@@ -18,6 +18,17 @@
         "provider": "${keycloak.realm.provider:}"
     },
 
+    "client": {
+        "provider": "${keycloak.client.provider:map}"
+    },
+
+    "mapStorage": {
+        "provider": "${keycloak.mapStorage.provider:concurrenthashmap}",
+        "concurrenthashmap": {
+            "dir": "${project.build.directory:target}"
+        }
+    },
```

Comment:
The JPA store should still be the default for developers. For those that are not working on the new store this would be a distraction and potential cause of failures. Personally for example I use KeycloakServer quite heavily, and often use it with a persisted H2 db.

I'd also suggest setting up a GitHub Actions workflow (or add a job to existing workflow) specifically for the new map store that.

Reply:
The store has been reverted to default to JPA for the developers, and GH Actions was added to test Undertow with map provider. Thanks for the suggestions!